source/javascripts/modules/navigation.js

```diff
@@ -0,0 +1,34 @@
+(function($, Modules) {
+  'use strict';
+
+  Modules.Navigation = function () {
+    var $html = $('html');
+
+    var $navToggle;
+    var $nav;
+
+    this.start = function ($element) {
+      $navToggle = $('.js-nav-toggle', $element);
+      $nav = $('.js-nav', $element);
+
+      updateAriaAttributes();
+
+      $navToggle.on('click', toggleNavigation);
+      $(window).on('resize', updateAriaAttributes);
```

Comment:
Unbounded listeners like this worry me but might be okay.

https://remysharp.com/2010/07/21/throttling-function-calls